ga4gh/datamodel/references.py

```diff
@@ -68,11 +69,13 @@ class SimulatedReferenceSet(AbstractReferenceSet):
     """
     A simulated referenceSet
     """
-    def __init__(self, id_):
+    def __init__(self, id_, randomSeed=0, numReferences=1):
         super(SimulatedReferenceSet, self).__init__(id_)
-        referenceId = "{}:srsone".format(id_)
-        reference = SimulatedReference(referenceId)
-        self._referenceIdMap[referenceId] = reference
+        for i in range(numReferences):
+            referenceId = "{}:srs{}".format(id_, i)
+            reference = SimulatedReference(referenceId, randomSeed)
```

Comment:
Does every Reference get the same random seed then? It would be good to give different references different seeds so they are not all identical. The easiest way to do that is to make a local random generator, seeded with randomSeed and use that to generate seeds for the Reference objects.